sql/sql_yacc.yy

```diff
@@ -5926,9 +5935,21 @@ create_table_option:
             $$= NEW_PTN PT_create_commen_option($3);
           }
         | COMPRESSION_SYM opt_equal TEXT_STRING_sys
-	  {
+          {
             $$= NEW_PTN PT_create_compress_option($3);
-	  }
+          }
+        | COMPRESSION_SYM opt_equal compression_types
```

Comment:
@hermanlee 
Is there a collision (at least logical) between InnoDB transparent page compression
`| COMPRESSION_SYM opt_equal TEXT_STRING_sys` (for instance `COMPRESSION='ZLIB'`, notice quotes here), which is a meaningful directive
and
`| COMPRESSION_SYM opt_equal compression_types` (for instance `COMPRESSION=ZLIB`, notice no quotes here), which should be ignored.
In my opinion, this is a bit confusing for the end user